CHANGELOG.md

```diff
@@ -0,0 +1,689 @@
+* **Inventory warning about unexepected keys**
+
+  An informative warning message is now logged when invalid group or node configuration keys are detected in the inventoryfile. ([BOLT-1017](https://tickets.puppet.com/browse/BOLT-1017))
+* **BoltSpec::Run support for uploading files to remote systems**
+
+  BoltSpec::Run now supports the upload_file action. ([BOLT-953](https://tickets.puppet.com/browse/BOLT-953))
+
+#### Bug fixes
+
+* **Remote tasks could run on non-remote targets**
+
+  Remote tasks can now be run only on remote targets ([BOLT-1203](https://tickets.puppet.com/browse/BOLT-1203))
+* **known_hosts weren't parsed correctly**
+
+  Previously, when a valid hostname entry was present in known_hosts and the host-key-check SSH configuration option was set, host key validation could fail when a valid IP address was not included in the known_hosts entry. This behavior was inconsistent with system SSH where the IP address is not required. Host key checking has been updated to match system SSH. ([BOLT-495](https://tickets.puppet.com/browse/BOLT-495))
+* **Plan variables were visible to sub-plans**
+
+  Variables defined in scope in a plan were visible to sub-plans called with run_plan. ([BOLT-1190](https://tickets.puppet.com/browse/BOLT-1190))
+
+## 1.14.0
+
+#### New features
+
+* **Support for Puppet device modules in a manifest block**
+
+  You can now apply Puppet code on targets that can't run a Puppet agent using the remote transport via a proxy. This is an experimental feature and might change in future minor (y) releases. ([BOLT-645](https://tickets.puppet.com/browse/BOLT-645))
+* **Validation and error handling for invalid PCP tokens**
+
+  The PCP transport token-file configuration option now includes validation and a more helpful error message. ([BOLT-1076](https://tickets.puppet.com/browse/BOLT-1076))
+
+## 1.13.1
+
+#### Bug fixes
+
+* **The \_run_as option was clobbered by configuration**
+
+  The run-as configuration option took precedence over the \_run_as parameter when calling run_* functions in a plan. The \_run_as parameter now has a higher priority than config or CLI. ([BOLT-1050](https://tickets.puppet.com/browse/BOLT-1050))
+* **Tasks with certain configuration options failed when using stdin**
+
+  When both interpreters and run-as were configured, tasks that required parameters to be passed over stdin failed. ([BOLT-1155](https://tickets.puppet.com/browse/BOLT-1155))
+
+## 1.13.0
+
+#### New features
+
+* **SMB file transfer on Windows**
+
+  When transferring files to a Windows host, you can now optionally use the SMB protocol to reduce transfer time. You must have either administrative rights to use an administrative share, like \\host\C$, or use UNC style paths to access existing shares, like \\host\share. You can use SMB file transfers only over HTTP, not HTTPS, and SMB3, which supports encryption, is not yet supported. ([BOLT-153](https://tickets.puppet.com/browse/BOLT-153))
```

Comment:
Picking the one below randomly, but it seems like it might be more effort than its worth to go through and add in places where there are `unformatted text` for all the old issues? 
```suggestion
 When transferring files to a Windows host, you can now optionally use the SMB protocol to reduce transfer time. You must have either administrative rights to use an administrative share, like `\\host\C$`, or use UNC style paths to access existing shares, like `\\host\share`. You can use SMB file transfers only over HTTP, not HTTPS, and SMB3, which supports encryption, is not yet supported. ([BOLT-153](https://tickets.puppet.com/browse/BOLT-153))
```

Reply:
I don't think it'd take very long, but sounds boring and I'm not sure if the value-add is worth the time it'd take to do it? Going forward we should definitely do that.

Reply:
Going to executive-decide not to do this.